waku/node/peer_manager/peer_manager.nim

```diff
@@ -75,6 +76,7 @@ type PeerManager* = ref object of RootObj
   switch*: Switch
   wakuPeerStore*: WakuPeerStore
   wakuMetadata*: WakuMetadata
+  wakuRelay*: WakuRelay
```

Comment:
This movement represents a quite big design change. We should keep all the protocols contained only within `Waku`